game-core/src/main/java/games/strategy/triplea/ai/pro/ProPurchaseAi.java

```diff
@@ -999,9 +999,8 @@ private void purchaseFactory(final Map<Territory, ProPurchaseTerritory> factoryP
       }
 
       // Determine units that can be produced in this territory
-      final List<ProPurchaseOption> purchaseOptionsForTerritory =
-          ProPurchaseUtils.findPurchaseOptionsForTerritory(player, purchaseOptions.getFactoryOptions(), maxTerritory,
-              isBid);
+      final List<ProPurchaseOption> purchaseOptionsForTerritory = ProPurchaseUtils
+          .findPurchaseOptionsForTerritory(player, purchaseOptions.getFactoryOptions(), maxTerritory, isBid);
```

Comment:
Just noting in case it wasn't your intention... this line is still calling the four-parameter version of `findPurchaseOptionsForTerritory()` where `t` == `factoryTerritory`.  Assuming you thought you were going to add the fifth parameter here but changed your mind and left the reformatted line in place.  Since the enclosing method is `purchaseFactory()`, I'm guessing that's a safe assumption. :smile:

Reply:
Yeah, that's exactly the case. I realized factoryOptions were only land territories so didn't need the fifth parameter.